src/lib/entities/content-type.ts

```diff
@@ -179,12 +180,20 @@ class EditorInterfaces {
     }
   }
 
-  addSidebarWidget (widgetId: string,
-                    widgetNamespace: APISidebarWidgetNamespace,
-                    insertBeforeWidgetId: string,
-                    settings: APISidebarWidgetSettings,
-                    disabled: boolean) {
-    this._sidebar = Array.isArray(this._sidebar) ? this._sidebar : []
+  addSidebarWidget (
+    widgetId: string,
+    widgetNamespace: APISidebarWidgetNamespace,
+    settings: APISidebarWidgetSettings,
+    insertBeforeWidgetId: string,
+    disabled: boolean
+  ) {
+
+    this._sidebar = Array.isArray(this._sidebar) ? this._sidebar : [].concat(DEFAULT_SIDEBAR_LIST)
```

Comment:
Why is that `[].concat` needed ? Isn't `DEFAULT_SIDEBAR_LIST ` already an array? 🤔

Reply:
Just in case to avoid mutation 😬